src/engine/sidechain/shoutconnection.cpp

```diff
@@ -611,12 +604,6 @@ bool ShoutConnection::processConnect() {
             kLogger.warning()
                     << "processConnect() socket error."
                     << "Is socket already in use?";
-        } else if (timeout >= kConnectRetries) {
-            // Not translated, because shout_get_error() returns also English only
-            m_lastErrorStr = QStringLiteral("Connection establishment time-out");
-            kLogger.warning()
-                    << "processConnect() error:"
-                    << m_iShoutStatus << m_lastErrorStr;
```

Comment:
why the removal?

Reply:
Oh that was a faulty rebase. Forced pushed the last commit.